tests/phpunit/tests/Story_Post_Type.php

```diff
@@ -304,6 +314,77 @@ public function test_add_caps_to_roles() {
 		}
 	}
 
+	/**
+	 * @covers ::redirect_post_type_archive_urls
+	 */
+	public function test_redirect_post_type_archive_urls_true() {
+		$story_post_type = new \Google\Web_Stories\Story_Post_Type( $this->createMock( \Google\Web_Stories\Experiments::class ) );
+		$query           = new \WP_Query();
+		$result          = $story_post_type->redirect_post_type_archive_urls( true, $query );
+		$this->assertTrue( $result );
+	}
+
+	/**
+	 * @covers ::redirect_post_type_archive_urls
+	 */
+	public function test_redirect_post_type_archive_urls_no_permalink() {
+		$story_post_type = new \Google\Web_Stories\Story_Post_Type( $this->createMock( \Google\Web_Stories\Experiments::class ) );
+		$query           = new \WP_Query();
+		$result          = $story_post_type->redirect_post_type_archive_urls( false, $query );
+		$this->assertFalse( $result );
+	}
+
+	/**
+	 * @covers ::redirect_post_type_archive_urls
+	 */
+	public function test_redirect_post_type_archive_urls_permalinks() {
+		global $wp_rewrite;
+		$wp_rewrite       = new \WP_Rewrite();
+		$wp_rewrite->init();
+		$wp_rewrite->permalink_structure = '/%post_name%/';
```

Comment:
```suggestion
		$this->set_permalink_structure( '/%post_name%/' );
```

Reply:
I did this originally, but it didn't work.